src/roles/roles.go

```diff
@@ -0,0 +1,14 @@
+package roles
+
+// Roles lists the roles
+var Roles = []string{"GE", "Grader"}
+
+func validateRole(role string) bool {
```

Comment:
This will need to be capitalized and documented, thanks to Go's magic public vs. private logic.  As it is right now, because it's lowercase, it isn't accessible outside the "roles" package.

Also, a best practice: keep the word "Role" out of functions.  The code would be called via `roles.ValidateRole(x)` if the function name is kept as-is.  A succinct name like `IsValid` or even just `Valid` would be clearer to read: `if roles.Valid("foo") { ... }`

Reply:
As an add-on the that second bit, you might be tempted to change the variable above from "Roles" to "All" or something in order to keep the word "Role" out.  That's a case that's a lot more subjective, it seems like.  There is a lot of code that repeats the package name because it's necessary and there just isn't a great way to describe the variable or function any other way.  This is rarely done with functions, hence my comment above, but it does happen with types and variables.

tl;dr: keep the variable named "Roles"